python/tvm/_ffi/_cython/base.pxi

```diff
@@ -27,7 +27,7 @@ cdef enum TVMTypeCode:
     kUInt = 1
     kFloat = 2
     kTVMOpaqueHandle = 3
-    kTVMNullptr = 4
+    kBFloat = 4
```

Comment:
sorry, we want to keep the type code for the normal argument as it is, and not changin the FFI

Reply:
shall we remove this?

Reply:
changed